app/views/shared/_header.html.erb

```diff
@@ -65,15 +65,23 @@
               <a class="dropdown-item" href="http://docs.bigbluebutton.org/install/greenlight-v2.html" target="_blank">
                 <i class="dropdown-icon far fa-question-circle"></i> <%= t("header.dropdown.help") %>
               </a>
-              <%= link_to logout_path, class: "dropdown-item" do %>
-                <i class="dropdown-icon fas fa-sign-out-alt"></i> <%= t("header.dropdown.signout") %>
+              <% if Rails.configuration.omniauth_auth0 %>
```

Comment:
As stated in the comment outside this review, the sign in and sign out have to be generalized